cst438_register/src/main/java/com/cst438/domain/StudentDTO.java

```diff
@@ -0,0 +1,29 @@
+package com.cst438.domain;
+
```

Comment:
I like how you coded a StudentDTO.java and StudentController.java

Reply:
Your new code is very clean. Giving a separate file is a fresh start and makes the review a lot more readable.